File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/SearchUserPresenter.java
package bobrchess.of.by.belaruschess.presenter;

import bobrchess.of.by.belaruschess.view.activity.impl.SearchUserActivity;

/**
 * Created by Igor on 12.04.2018.
 */

public interface SearchUserPresenter extends BasePresenter{
    void loadUsers();

    void loadUsers(Integer count);

    void searchUsers();

    void attachView(SearchUserActivity activity);
}

File: app/src/main/java/bobrchess/of/by/belaruschess/presenter/SearchUserPresenter.kt
package bobrchess.of.by.belaruschess.presenter

import bobrchess.of.by.belaruschess.view.activity.SearchUserContractView

/**
 * Created by Igor on 12.04.2018.
 */

interface SearchUserPresenter : BasePresenter {
    fun loadUsers()

    fun loadUsers(count: Int?)

    fun searchUsers(text: String)

    fun attachView(searchUserContractView: SearchUserContractView)
}
